app/helpers/site_title_helper.rb

```diff
@@ -0,0 +1,5 @@
+module SiteTitleHelper
+  def site_title
+    Setting.site_title.presence
```

Comment:
Good catch, that was from an earlier version which made this more complicated than it had to be before I understood what `Setting` was doing. Updated to just ensure a string is there.